g11n-ws/tools/tool-trans-fetcher/src/main/java/com/vmware/vip/fetcher/common/FileUtils/OrderedPropertiesUtils.java

```diff
@@ -37,8 +37,8 @@ public static OrderedProperties loadFromFile(String fileName) {
         OrderedProperties p = new OrderedProperties();
 
         try {
-            InputStream inputStream = new FileInputStream(fileName);
-            p.load(new InputStreamReader(inputStream, ConstantsUnicode.UTF8));
+          
+            p.load(new InputStreamReader(new FileInputStream(fileName), ConstantsUnicode.UTF8));
```

Comment:
Where to close the stream?

Reply:
this doesn't need to close, it is a parameter not variable